resource-timing/resource_TAO_multi.htm

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8" />
+<title>Resource Timing TAO tests</title>
```

Comment:
Are we testing L1 or L2? The definitions differ, see discussion in https://github.com/w3c/resource-timing/issues/62.